appveyor.yml

```diff
@@ -1,15 +0,0 @@
-# This is a dummy configuration file that *does not run any test*.
-# The file should be deleted, and Appveyor deactivated as soon as all
-# opened PRs have this configuration merged in.
-# SEE PR#2908 (https://github.com/yt-project/yt/pull/2908/files)
-
-platform:
-    - x64
-
-install:
-    - "echo 'Not installing anything.'"
-
-build: false
-
-test_script:
-    - "echo 'Not testing anything. This is done on Travis'"
```

Comment:
Deleting the `appveyor.yml` file will make this PR and all subsequent one fail until we deactivate appveyor completely. I would suggest delaying this to a future PR (otherwise I think we'd have to update all existing PRs to remove the appveyor.yml file).

Reply:
+1 to this. I think we should have dummy appveyor and travis files like what corentin already did with appveyor. What if we added a comment in both files that they'll be removed officially from master in ~3-6 months (and the date can be recorded in the file maybe) or something?

Reply:
There's always going to be a point when there are outstanding PRs and removing this file on master will force them to merge/rebase. I don't think setting a date will make a difference. As for appveyor I was planning on disabling it even before this goes in.

Reply:
Also I don't think anyone will need to remove appveyor.yml, unless they made some changes locally that were not on master.

Reply:
> There's always going to be a point when there are outstanding PRs and removing this file on master will force them to merge/rebase. 

Yes that's true. The reason I proposed this is because we've asked contributors to do this several times this year, which hasn't been the case in the past. I could see that be frustrating or even offputting.